### ISSUE TITLE
testing in firefox

### ISSUE THREAD

**Issue by wcornwell** [coreytcallaghan/eBirder-visualizations#24]
nodes and wikipedia aren't working for me in firefox atm

**Comment by johnWilshire** [comment]
Good catch, I am not sure what is causing this but I think it is something in the async computed.

I cant find anything about it in a quick google so I will write a small wiki scraper so we dont have to rely on async requests and can just have all the info box's populated from data that we send to the client.

**Comment by johnWilshire** [comment]
It would mean that we have to run the scraper again each time we change the trees, it should be fine though. 

This solution will work for the small number of trees that we have but it will not scale to a large number of trees
